week02/이성진/todo/src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import './App.css'
+import TodoBoard from './component/TodoBoard'
+
+function App() {
+  const [inputValue,setInputValue] = useState('')
+  const [todolist,setTodolist]=useState([])
+  const addItem =() => {
+    console.log("content",inputValue)
+    setTodolist([...todolist,inputValue])
+  }
+  return (
+    
+    <div className='mainBoard'>
+      <div className='main'>
+        <h1>TODOLIST</h1>
+        <input className='input' value={inputValue} type="text" placeholder='해야할 일을 입력하세요' onChange=
+        {(event)=> setInputValue(event.target.value)}/> 
```

Comment:
```suggestion
        <input 
          className='input' 
          value={inputValue} 
          type="text" 
          placeholder='해야할 일을 입력하세요' 
          onChange={(event) => setInputValue(event.target.value)}/> 
```

해당 코드처럼 가독성을 챙기는 습관도 들이면 좋겠어요

Reply:
수정하겠습니다!